PocketKit/Sources/PocketGraph/user-defined-operations/itemFragments.graphql

```diff
@@ -2,26 +2,34 @@ fragment ItemParts on Item {
   remoteID: itemId
   givenUrl
   resolvedUrl
-  title
   language
-  topImageUrl
   timeToRead
-  domain
-  datePublished
   isArticle
   hasImage
   hasVideo
   wordCount
-  authors {
-    id
-    name
-    url
-  }
   collection {
```

Comment:
Do we still need this?

Reply:
we need the slug to determine if it's a collection, we could remove the others, but, fixtures...

Reply:
gotcha